### PR TITLE
Router link

### PR DESCRIPTION
**Pull request by GeorgesFORTRY**
creation of links to the pages of the app

### CODE REVIEW

**Review by JulienDelemarle — APPROVED** [review]
routers: check ok.

**Comment by LoicBrassart** [comment]
(Reminder: It's the SM who should assign me to the PR ;-) )

File: package.json [hunk 1]
```diff
@@ -7,6 +7,8 @@
     "@testing-library/jest-dom": "^5.11.10",
     "@testing-library/react": "^11.2.6",
     "@testing-library/user-event": "^12.8.3",
+    "axios": "^0.21.1",
```
Comment (LoicBrassart) [inline package.json:10]: Why add axios now ?

**Review by LoicBrassart — APPROVED** [review]
Saw no real Evil

**Pull request merged**